### DISCUSSION TITLE
Missing strings

### DISCUSSION THREAD

**Discussion by derekbreth** [DotSpatial/DotSpatial#1490] [General]
We the projections library which is generally excellent however we get 'Projection not found' often with local South African projections when using DotSpatial.Projections.ProjectionInfo.FromEsriString. Is there an easy way we can add the missing projection strings?

**Comment by jany-tenaj** [comment]
you could submit a pull request in which you added the missing projections to the [Africa.cs](https://github.com/DotSpatial/DotSpatial/blob/master/Source/Core/DotSpatial.Projections/ProjectedCategories/Africa.cs)
